.github/workflows/rule-validate.yml

```diff
@@ -35,21 +35,43 @@ jobs:
         uses: alessbell/pull-request-comment-branch@v1.1 # Fork of xt0rted/pull-request-comment-branch, see https://github.com/xt0rted/pull-request-comment-branch/issues/322
         id: comment_branch
 
-      - name: Checkout (from comment)
-        uses: actions/checkout@v3
-        if: github.event_name == 'issue_comment'
-        with:
-          repository: ${{ steps.comment_branch.outputs.head_owner }}/${{ steps.comment_branch.outputs.head_repo }}
-          ref: ${{ steps.comment_branch.outputs.head_ref }}
-          fetch-depth: 0
+      - name: Get Refs
+        id: get_refs
```

Comment:
I refactored this to handle some more logic. When a comment was triggering, we were running MQL Mimic with a ref of `main`, not the PR branch (see line 302 of the old code)